test/e2e/webhook_test.go

```diff
@@ -0,0 +1,54 @@
+package e2e
+
+import (
+	"strconv"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	nmstatev1alpha1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1alpha1"
+	mutatingwebhook "github.com/nmstate/kubernetes-nmstate/pkg/webhook/mutating"
+)
+
+var _ = Describe("Mutating Admission Webhook", func() {
+	policyName := "webhook-test"
```

Comment:
Why do we have a dedicated name here, I thought we already have one used everywhere.

Reply:
This test check that we catch CREATE and UPDATE policy events at webhook, so we have to be sure that policy does not exist that's why I create a new one, but we can ensure that TestPolicy is deleted at BeforeEach and use it.

Reply:
I think that using the same name and ensuring that it is not present is more expressive.